bastion/config/local.yaml

```diff
@@ -0,0 +1,2 @@
+provider_url: "https://eth-goerli.g.alchemy.com/v2/RWhchAQNylFZLnnPO7Rmj3b4T4uZIFuO"
```

Comment:
is this for local test or dev test?

Reply:
Forgot to put it in gitignore since I added later. Just added it to gitignore.